content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```diff
@@ -558,3 +558,178 @@ while (resultSet.hasNext()) {
 ```
 {{% /tab %}}
 {{< /tabs >}}
+
+# Vault-Named Query Operators
+
+The following is the list of the operators and right operand types for the vault-named query syntax:
+
+* `IN`
+* `LIKE`
+* `IS NOT NULL`
+* `IS NULL`
+* `AS`
+* `OR`
+* `AND`
+* `!=`
+* `>`
+* `<`
+* `>=`
+* `<=`
+
+### Detailed Operators
```

Comment:
```suggestion
### Detailed Operators
```
Is this necessary?

Reply:
I added the second header to distinguish that only some operators had descriptions and not the entire list. Do you think it would be better to not have it?

Reply:
This relates to my other comment. I think if you had a full list, with links where applicable, this heading isn't necessary.

Could add maybe add a sentence such as the following:

Where the behaviour is not standard, the operators are explained in detail in the following sections..